src/vision.cs

```diff
@@ -59,7 +60,7 @@ enum VNRequestTrackingLevel : ulong {
 	enum VNImageCropAndScaleOption : ulong {
 		CenterCrop = 0,
 		ScaleFit = 1,
-		ScaleFill,
+		ScaleFill = 2,
```

Comment:
Why isn't this getting an availability attribute?

Reply:
Because it is not needed just matching headers value which is the same implicit value

Reply:
Oh yea thought it was new like the one I commented above 👍